netket/optimizer/qgt/qgt_jacobian_pytree.py

```diff
@@ -96,7 +96,6 @@ def _solve(self, solve_fun, y: PyTree, *, x0: Optional[PyTree] = None) -> PyTree
         if self.scale is not None:
             grad = grad / self.scale
 
-        solve_fun = self.sr.solve_fun()
```

Comment:
Could you undo this change to make #689 mergeable?

Reply:
let me try (no guarantees)

Reply:
ok it worked